apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceConfiguration.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.Diagnostics.Common;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// Optional configuration to be used when initializing tracing.
+    /// </summary>
+    public sealed class TraceConfiguration
```

Comment:
Is this file meant to be in this PR, or did I mess up somehow? It doesn't seem to be referenced anywhere else.